packages/core/bootstrap/src/lib/util.ts

```diff
@@ -373,3 +373,101 @@ export const getRequiredEnvWithFallback = (
 
   throw new RequiredEnvError(getEnvName(primary, prefix))
 }
+
+//  URL Encoding
+
+const charsToEncode = {
+  ':': '%3A',
+  '/': '%2F',
+  '?': '%3F',
+  '#': '%23',
+  '[': '%5B',
+  ']': '%5D',
+  '@': '%40',
+  '!': '%21',
+  $: '%24',
```

Comment:
Any reason this isn't in quotes when the others are?

```suggestion
  '$': '%24',
```

Reply:
It looks like the quotes are being removed by Prettier. I checked and it looks like the dollar sign is successfully used as a key, even without the quotes. I can investigate further if you think this could be an issue

Reply:
This would be why (apparently): https://mothereff.in/js-properties